contrib/opencensus-ext-azure/tests/test_azure_log_exporter.py

```diff
@@ -78,6 +78,12 @@ def test_ctor(self):
         self.assertRaises(ValueError, lambda: log_exporter.AzureLogHandler())
         Options._default.instrumentation_key = instrumentation_key
 
+    def test_invalid_sampling_rate(self):
+        self.assertRaises(ValueError, lambda: log_exporter.AzureLogHandler(
```

Comment:
FWIW you can do `with self.assertRaises(ValueError) as err` to avoid the lambda here.